### PR TITLE
Keymap changes

### PR DESCRIPTION
**Pull request by theappleman**
Based on my current usage, I make the following changes:
- LH L0 Backslash, 5 letters across the bottom
- TC L0 Switch backspace to RH
- RH L0 Symbol changes, puts ,./ on the same line. Colon for vim next to L key.
- RH L0 Layer toggles switch (break symmetry)
- RH L1 Move zero to left of dot

### CODE REVIEW

File: keyboard/ergodox_ez/keymaps/ben/keymap.c [hunk 1]
```diff
@@ -16,41 +16,41 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
  * |--------+------+------+------+------+-------------|           |------+------+------+------+------+------+--------|
- * | Tab    |   Q  |   W  |   E  |   R  |   T  |  Y   |           |  L1  |   Y  |   U  |   I  |   O  |   P  | Del    |
+ * | Tab    |   Q  |   W  |   E  |   R  |   T  |  Y   |           |  L2  |   Y  |   U  |   I  |   O  |   P  | Del    |
  * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
- * | Tab    |   A  |   S  |   D  |   F  |   G  |------|           |------|   H  |   J  |   K  |   L  | '/@  | Enter  |
- * |--------+------+------+------+------+------|  L2  |           |  L2  |------+------+------+------+------+--------|
- * | LShift | \/|  |   Z  |   X  |   C  |   V  |      |           |      |   N  |   M  | #/~  | //?  | ;/:  | RShift |
+ * | Tab    |   A  |   S  |   D  |   F  |   G  |------|           |------|   H  |   J  |   K  |   L  | ;/:  | Enter  |
+ * |--------+------+------+------+------+------|  L2  |           |  L1  |------+------+------+------+------+--------|
+ * | LShift |   Z  |   X  |   C  |   V  |   B  |      |           |      |   N  |   M  | ,/<  |  ./> | //?  | RShift |
```
Comment (bencord0) [inline keyboard/ergodox_ez/keymaps/ben/keymap.c:23]: I'm not too sure about the LH.
Based on my own usage, zxcv should stay. I'll entertain moving B to the L2 position.

File: keyboard/ergodox_ez/keymaps/ben/keymap.c [hunk 2]
```diff
@@ -16,41 +16,41 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  * ,--------------------------------------------------.           ,--------------------------------------------------.
  * | ESC    |  1/! |  2/" |  3/£ |  4/$ |  5/% |  6/^ |           |  7/& |  8/ *|  9/( |  0/) |  -/_ | =/+  | BkSp   |
  * |--------+------+------+------+------+-------------|           |------+------+------+------+------+------+--------|
- * | Tab    |   Q  |   W  |   E  |   R  |   T  |  Y   |           |  L1  |   Y  |   U  |   I  |   O  |   P  | Del    |
+ * | Tab    |   Q  |   W  |   E  |   R  |   T  |  Y   |           |  L2  |   Y  |   U  |   I  |   O  |   P  | Del    |
```
Comment (bencord0) [inline keyboard/ergodox_ez/keymaps/ben/keymap.c:19]: Why switch L1/L2 on the RH?

File: keyboard/ergodox_ez/keymaps/ben/keymap.c [hunk 3]
```diff
@@ -16,41 +16,41 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
  * `--------+------+------+------+------+-------------'           `-------------+------+------+------+------+--------'
- *   |  ~L1 |      | Cmd  | Alt  | Ctrl |                                       |   B  | ,/<  | ./>  | Alt  | Ctrl   |
+ *   |  ~L1 | \/|  | Cmd  | Alt  | Ctrl |                                       |   B  | '/@  | #/~  | Alt  | Ctrl   |
  *   `----------------------------------'                                       `------------------------------------'
  *                                        ,-------------.       ,-------------.
  *                                        | Home | End  |       | Left | Right|
  *                                 ,------|------|------|       |------+--------+------.
  *                                 |      |      | PgUp |       | Up   |        |      |
- *                                 | Space| BkSp |------|       |------|  Space | Space|
+ *                                 | Space| Space|------|       |------|  Bksp  | Space|
```
Comment (bencord0) [inline keyboard/ergodox_ez/keymaps/ben/keymap.c]: Tempted to make these all space. I don't use backspace here at all.

Comment (theappleman) [inline keyboard/ergodox_ez/keymaps/ben/keymap.c]: Yeah, it is certainly dangerous here. Useful, but dangerous.

File: keyboard/ergodox_ez/keymaps/ben/keymap.c (hunk 2 above)
Comment (theappleman) [inline keyboard/ergodox_ez/keymaps/ben/keymap.c:19]: No particular reason. L1 toggle would be closer to a thumb, like L2 toggle on the LH.

**Comment by theappleman** [comment]
closing

**Pull request closed without merging**